src/main/scala/com/github/unchama/seichiassist/listener/SpawnRegionProjectileInterceptor.scala

```diff
@@ -1,31 +1,38 @@
 package com.github.unchama.seichiassist.listener
 
-import com.github.unchama.util.external.WorldGuardWrapper.{getOneRegion, getRegions}
-import org.bukkit.entity.Player
-import org.bukkit.event.entity.ProjectileLaunchEvent
+import com.github.unchama.util.external.WorldGuardWrapper.getRegions
+import org.bukkit.Material._
+import org.bukkit.event.block.Action
+import org.bukkit.event.player.PlayerInteractEvent
 import org.bukkit.event.{EventHandler, Listener}
 
 import scala.jdk.CollectionConverters._
 
 object SpawnRegionProjectileInterceptor extends Listener {
-  @EventHandler
-  def onProjectileLaunch(event: ProjectileLaunchEvent): Unit = {
-    val projectile = event.getEntity
-    if (projectile == null) return
+  val spawnRegions = Set(
+    // 基本の保護名
+    "spawn",
+    // メインワールドにおいて、スポーン地点を保護している保護名
+    "spawn-center",
+    // 公共施設サーバーのスポーン地点名
+    "world-spawn"
+  )
 
-    val spawnRegions = Set(
-      // 基本の保護名
-      "spawn",
-      // メインワールドにおいて、スポーン地点を保護している保護名
-      "spawn-center",
-      // 公共施設サーバーのスポーン地点名
-      "world-spawn"
+  @EventHandler
+  def beforeProjectileLaunch(event: PlayerInteractEvent): Unit = {
+    val player = event.getPlayer
+    val inventory = player.getInventory
+    val action = event.getAction
+    val projectiles = Set(
+      BOW, EGG, LINGERING_POTION, SPLASH_POTION, ENDER_PEARL, EYE_OF_ENDER, FIREBALL, SNOW_BALL, EXP_BOTTLE
```

Comment:
```suggestion
      BOW, EGG, LINGERING_POTION, SPLASH_POTION, ENDER_PEARL, EYE_OF_ENDER, SNOW_BALL, EXP_BOTTLE
```
Materialでの`FIREBALL`は`minecraft:fire_charge`であり、Minecraftの仕様上右クリックでの投擲が不可能であるため、必要ないのではないでしょうか？

Reply:
コメントありがとうございます。確かにプレイヤーが直接発射することはできないので、該当suggestionをコミットします。